### ISSUE TITLE
Why is the package org.jfree.chart.fx excluded from the maven package?

### ISSUE THREAD

**Issue by xylo** [jfree/jfreechart#3]
Would ChartCanvas and ChartView not be useful for the public?

**Comment by jfree** [comment]
Historically JFreeChart has supported JDK1.6 or later, so I didn't want to break that for the default build.  To use the JavaFX features, you currently have to build with a specific Ant script and JDK 1.8.0.  In the next release, I will include a Maven build for the JavaFX code, and ensure a separate artifact is available on The Central Repository.

**Comment by xylo** [comment]
Thanks for the info.

**Comment by simon04** [comment]
Could the `org.jfree.chart.fx` package be moved to a separate GitHub project and separate Maven artifact?

**Comment by jfree** [comment]
That's an option.  My current thinking is to build version 1.0.20 using Java 6 and exclude the JavaFX code, and version 1.5.0 with Java 8 and include the JavaFX code...both from the same repo (just switching pom files as required).

**Comment by jfree** [comment]
See the alternate pom files in the 'maven' folder.

**Comment by gforman44** [comment]
I like your idea of separate 1.5.0      I can't wait for it.  (figuratively and literally)

**Comment by jfree** [comment]
I'm going to make a big effort (but no promises) to get this done before the end of Dec.

**Comment by gforman44** [comment]
David,
December's a mighty busy month with Christmas and all, but if you *do* get it done, it'll be a Christmas present to developers.
(My workaround was to compile the *fx/ source code separately in Eclipse for now.  Thus, for me, the pressure is off.  But I thought I'd let you know there are developers out there who care.)
BTW, I find JFreeChart so much faster than JavaFx charts.   I had to make data subsetter classes to use JavaFx charts with audio waves.  All that animation stuff...
Thank you so much for making JFreeChart.  I would have wished that the value you brought to the world by it would compensate you nicely, but I've heard the story that selling good documentation was not a win, ultimately.  I bought a copy or two while I worked at HP.
   George
 

    On Monday, December 5, 2016 12:55 PM, David Gilbert <notifications@github.com> wrote:
 

 I'm going to make a big effort (but no promises) to get this done before the end of Dec.—
